dockstore-webservice/src/main/resources/migrations.1.10.0.xml

```diff
@@ -74,4 +74,12 @@
             <column name="licensename" type="varchar(255 BYTE)"/>
         </addColumn>
     </changeSet>
+    <changeSet id="changeOrganizationUserRoles" author="natalieperez">
```

Comment:
We need to ensure that the old web service is not running after this, otherwise there's the chance for another one to slip in. We need to update the 1.10 (1.9.x?) release checklist, but I haven't really thought about how we're going to handle migrations.